### PR TITLE
New release 4.5.3

### LINKED ISSUE #2: Package out of date
There a new [release](https://downloads.airtame.com/app/latest/linux/Airtame-4.5.3.deb) v4.5.3 available, but the deb package is no longer linked publicly on their website.

Might be worth considering using their snap package as a base if possible as it seems to currently be the only officially listed / supported format.

### PR DESCRIPTION
**Pull request by bbhtt**
Closes https://github.com/flathub/com.airtame.Client/issues/2

Took the release date from here https://help.airtame.com/hc/en-us/articles/5308029216157-Install-the-Airtame-app-on-Linux- (the date it was updated on) because they don't mention it on https://help.airtame.com/hc/en-us/articles/5348392729117-Changelog-Product-Changes-Airtame-Application-

### CODE REVIEW

File: com.airtame.Client.appdata.xml [hunk 1]
```diff
@@ -26,6 +26,7 @@
   </categories>
   <!-- https://help.airtame.com/hc/en-us/articles/5348392729117-Changelog-Product-Changes-Airtame-Application- -->
   <releases>
+    <release version="4.5.3" date="2022-11-15" />
```
Comment (klausenbusk) [inline com.airtame.Client.appdata.xml]: My best guess is that it was released on `2022-08-12`. https://snapcraft.io/airtame-application was last updated on that date and the last-modified of `https://downloads.airtame.com/app/latest/linux/Airtame-4.5.3.snap` is also that date.

Perhaps you could add the changelog for 4.5.0 and 4.5.2 as documented [here](https://help.airtame.com/hc/en-us/articles/5348392729117-Changelog-Product-Changes-Airtame-Application-)?

Comment (bbhtt) [reply com.airtame.Client.appdata.xml]: 4.5.2 is on 2022-09-12 per the changelog, I don't think 4.5.3 is 2022-08-12? Anyways I added the it and the changelogs.

File: com.airtame.Client.appdata.xml [hunk 2]
```diff
@@ -26,6 +26,37 @@
...
+        <p>New features</p>
+          <ul>
+            <li>802.1X authentication for ethernet added in setup flow and device settings</li>
+            <li>Rooms: Sign in with Google Calendar to have easier access to your meetings</li>
+            <li>Rooms: Added the ability to join calls on Linux</li>
+          </ul>
+        <p>Improvements</p>
+          <ul>
+            <li>New streaming technology for Linux: WebRTC-based streaming to provide a smoother screen sharing experience and the ability to select which screen to share in multi-screen setups. This also adds support for the latest Ubuntu LTS version</li>
+            <li> Audio on Linux is for the moment no longer supported. We will seek to add audio at a later date. Customers who wish to use audio streaming on Linux should stay on app version 4.1.1</li>
```
Comment (klausenbusk) [inline com.airtame.Client.appdata.xml]:
```suggestion
            <li>Audio on Linux is for the moment no longer supported. We will seek to add audio at a later date. Customers who wish to use audio streaming on Linux should stay on app version 4.1.1</li>
```

File: com.airtame.Client.appdata.xml [hunk 3]
```diff
@@ -26,6 +26,37 @@
...
+            <li>Added new security controls following Electron best practices, including but not limited to the prevention of untrusted resources</li>
+          </ul>
+        <p>Bug fixes</p>
+          <ul>
+            <li>Rooms: Fixed an issue that prevented sharing a single window to a Teams call when two overlapping windows of the same program are present, e.g. during a PowerPoint presentation</li>
+            <li>Rooms: Fixed an issue that prevented sharing content from a secondary monitor to a Teams call</li>
+            <li> Rooms: Cancelling a request to join a call in the Airtame application should now properly return the Airtame Hub to an idle state</li>
```
Comment (klausenbusk) [inline com.airtame.Client.appdata.xml]:
```suggestion
            <li>Rooms: Cancelling a request to join a call in the Airtame application should now properly return the Airtame Hub to an idle state</li>
```

File: com.airtame.Client.appdata.xml (hunk 2 above)
Comment (bbhtt) [reply com.airtame.Client.appdata.xml]: Thanks.

**Comment by klausenbusk** [comment]
Sorry for being utter slow :)

**Pull request merged**